lib/sprockets/loader.rb

```diff
@@ -186,7 +186,7 @@ def load_from_unloaded(unloaded)
           dependencies_digest: DigestUtils.digest(resolve_dependencies(metadata[:dependencies]))
         }
 
-        asset[:id]  = pack_hexdigest(digest(asset))
```

Comment:
Actually maybe we can re-use asset ID which should already be a hexdigest. I'll have to look into it.